stetho/src/main/java/com/facebook/stetho/inspector/database/DatabasePeerManager.java

```diff
@@ -30,18 +29,43 @@
   };
 
   private final Context mContext;
+  private final DatabaseFilesProvider mDatabaseFilesProvider;
 
-  public DatabasePeerManager(Context context) {
+  /**
+   * Constructs the object with a {@link DatabaseFilesProvider} that supplies the database files
+   * from {@link Context#databaseList()}.
+   *
+   * @param context the context
+   */
+  public DatabasePeerManager(final Context context) {
+    this(context, new DatabaseFilesProvider() {
```

Comment:
I'm not a fan of having the default applied internally.  Instead, create a `DefaultDatabaseFilesProvider` class and modify all constructors to require an instance of `DatabaseFilesProvider`.  Then, the default will be to merely invoke:
```java
new Database(context, new DefaultDatabaseFilesProvider(context));
```